clusters/app.ci/registry-access/ocp-qe/admin_manifest.yaml

```diff
@@ -0,0 +1,75 @@
+# this is the Namespace in which your ServiceAccount will live
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: Automation ServiceAccounts for OCP QE
+    openshift.io/display-name: OCP QE
+  name: ocp-qe
+---
+# this is the ServiceAccount whose credentials you will use
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: image-puller
+  namespace: ocp-qe
+---
+# this grants your ServiceAccount rights to pull images
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: ocp-qe-image-puller-binding
+  # the namespace from which you will pull images
+  namespace: ocp
+roleRef:
+  kind: ClusterRole
+  apiGroup: rbac.authorization.k8s.io
+  name: system:image-puller
+subjects:
+  - kind: ServiceAccount
+    namespace: ocp-qe
+    name: image-puller
+---
+# the Group of people who should be able to manage this ServiceAccount
+kind: Group
+apiVersion: v1
+metadata:
+  name: ocp-qe-admins
+users:
+  # these names are GitHub usernames
+  - akostadinov
+  - jianlinliu
+  - pruan-rht
+---
+# this grants the right to read the ServiceAccount's credentials
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: secret-reader
+  namespace: ocp-qe
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - serviceaccounts
+      - secrets
+    verbs:
+      - get
+      - list
```

Comment:
Can you please update this to the current form of the https://docs.ci.openshift.org/docs/how-tos/use-registries-in-build-farm/#how-do-i-get-a-token-for-programmatic-access-to-the-central-ci-registry doc?

Individual teams no longer need to define this `secret-reader` role themselves and should just use the provided `pull-secret-namespace-manager` cluster role.

Reply:
Thanks for your comments, updated it accordingly.

Reply:
You did not, the rolebinding still references the `secret-reader`

Reply:
Oops, my fault, amended it, pls help review again.